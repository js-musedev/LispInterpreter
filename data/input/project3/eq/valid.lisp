(EQ 23 23)
(EQ (EQ 34 34) (EQ (EQ 23 23) NIL))
(EQ T T)
(EQ (INT NIL) (NULL T))
(INT (EQ 34 34))
