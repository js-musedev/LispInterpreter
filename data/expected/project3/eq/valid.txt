T
NIL
T
T
NIL